react-frontend/src/components/Nav/navbar.js

```diff
@@ -1,7 +1,11 @@
 import React, { useState, useEffect } from 'react';
+import { Drawer, Menu } from 'antd';
+import '../../css/nav.css';
```

Comment:
This file no longer exists, the project is not able to run due to this.

Reply:
Merge artifact. Removing this.